index.html

```diff
@@ -7,6 +7,6 @@
     <title>Hello Microverse Project</title>
```

Comment:
- [ ] As stated in the requirement, and as I can see that you have your `style.css` file; the only challenge is that you forgot to link your `index.html` with your `style.css` so that the `color` property can take effect. Kindly link the file in your `head` section here.

_An example of how to link is:_
```
<link rel="stylesheet" href="./style.css">
```
That should make everything work well :smile: